gcp_variant_transforms/beam_io/vcf_parser.py

```diff
@@ -802,10 +601,12 @@ def _get_variant_calls(self, samples):
           phaseset = (
               list(map(self._convert_field, value, [False] * (len(value)))) if
               isinstance(value, tuple) else
-              self._convert_field(value, numeric=False))
+              self._convert_field(value, False))
         else:
-          info[key] = (list(map(self._convert_field, value)) if
-                       isinstance(value, tuple) else self._convert_field(value))
+          info[key] = (self._homogenize_list_dtype(
+              map(self._convert_field, value, [True] * (len(value)))) if
+                       isinstance(value, tuple) else
+                       self._convert_field(value, True))
 
       # PySam samples are "phased" for haploids, so check for for the type
       # before settings default phaseset value.
```

Comment:
In the following if statement, do we need the last condition `len(genotype) > 1`? It seems `sample.phased` includes that condition:
https://github.com/pysam-developers/pysam/blob/430c883ecd5f04080d00aa950ebd259aa38f40bc/pysam/libcbcf.pyx#L3420